ci-operator/step-registry/rosa/lifecycle/upgrade/rosa-lifecycle-upgrade-workflow.yaml

```diff
@@ -0,0 +1,17 @@
+workflow:
```

Comment:
Why we need a new workflow?

Reply:
If it is frequently reused, can we schedule the upgrade before nodes getting ready? It can save time.

Reply:
updated it